src/datasets/io/handler.py

```diff
@@ -1,7 +1,9 @@
+import bz2
 import gzip
+import lzma
 
 
-class IOHandler:
+class FileWriteHandler:
```

Comment:
@lhoestq I've changed IOHandler to FileWriteHandler as I realised technically we are only writing files (compressed/uncompressed) and not reading them here. What do you think about it?

Reply:
Sounds good !